### PR TITLE
Quieter test logs.

### PR DESCRIPTION
**Pull request by xian**

### CODE REVIEW

**Comment by jaredsburrows** [comment]
👍

File: robolectric/src/test/java/org/robolectric/shadows/SQLiteDatabaseTest.java [hunk 1]
```diff
@@ -910,4 +915,16 @@ public void shouldGetBlobFromString() {
         data.moveToFirst();
         assertThat(data.getBlob(0)).isEqualTo(values.getAsString("first_column").getBytes());
     }
+
+    /////////////////////
+
+    private SQLiteDatabase openOrCreateDatabase(String name) {
```
Comment (jongerrish) [inline robolectric/src/test/java/org/robolectric/shadows/SQLiteDatabaseTest.java]: Could these be created as memory backed databases for faster test performance?

https://developer.android.com/reference/android/database/sqlite/SQLiteDatabase.html#create(android.database.sqlite.SQLiteDatabase.CursorFactory)

Comment (xian) [inline robolectric/src/test/java/org/robolectric/shadows/SQLiteDatabaseTest.java]: Most likely, I'll try.

Comment (xian) [inline robolectric/src/test/java/org/robolectric/shadows/SQLiteDatabaseTest.java]: Not super straightforward; `android.database.sqlite.SQLiteException: Cannot prepare statement, base error code: 1`, whatever that means.

**Pull request merged**